backend/api/Cargo.toml

```
@@ -18,6 +18,8 @@ futures = "0.3.1"
 juniper = "0.15"
 log = "0.4"
 paste = "1"
+postgres-derive = "0.4"
+postgres-types = "0.1.3"
```

```suggestion
postgres-types = { version = "0.1.3", features = ["derive"] }
```

Seems cleaner to me. According to its own docs, `postgres-derive` is an internal crate.

---

Oh, I didn't know that.